src/js/components/InfiniteScroll/InfiniteScroll.js

```diff
@@ -196,9 +243,15 @@ const InfiniteScroll = ({
 
   const result = [];
 
+  const avgPageHeight =
```

Comment:
We are duplicating this calculation. Does it make sense to pull into a single calc?

Reply:
i can do that